README.md

```diff
@@ -169,20 +201,9 @@ eslint --fix --ext .js,.vue src
 ```
 
 ## How to Get Involved?
-
 moja global welcomes a wide range of contributions as explained in [Contributing document](https://github.com/moja-global/About-moja-global/blob/master/CONTRIBUTING.md) and the [About moja-global Wiki](https://github.com/moja-global/.github/wiki).
 
-## FAQ and Other Questions
-
-- You can find FAQs on the [Wiki](https://github.com/moja.global/.github/wiki).
-- If you have a question about the code, submit [user feedback](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Provide-User-Feedback.md) in the relevant repository
-- If you have a general question about a project or repository or moja global, [join moja global](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md) and
-  - [submit a discussion](https://help.github.com/en/articles/about-team-discussions) to the project, repository or moja global [team](https://github.com/orgs/moja-global/teams)
-  - [submit a message](https://get.slack.help/hc/en-us/categories/200111606#send-messages) to the relevant channel on [moja global's Slack workspace](mojaglobal.slack.com).
-- If you have other questions, please write to info@moja.global
-
```

Comment:
Could you please bring this part back again? Sorry, I should have mentioned it earlier. My bad.